packages/bsky/src/auth-verifier.ts

```diff
@@ -134,12 +134,29 @@ export class AuthVerifier {
           iss: null,
           aud: null,
         })
-        if (!opts.skipAudCheck && !this.standardAudienceDids.has(aud)) {
+
+        const [audDid, audServiceId] = aud.split('#')
+
+        if (!opts.skipAudCheck && !this.standardAudienceDids.has(audDid)) {
```

Comment:
We'd only like to accept one new audience: `ownDid + '#bsky_appview'`, but this is a bit looser than that.